fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -59,14 +62,69 @@
 
   private final Id getPermissionCounterId;
 
+  private final RetryHandler retryHandler;
+
+
+  interface RetryHandler {
+    default <T> T retry(String description, Callable<T> callable) throws Exception {
+      return callable.call();
+    }
+
+    RetryHandler NOOP = new RetryHandler() {};
+  }
+
+  static class ExponentialBackoffRetryHandler implements RetryHandler {
+    private final long maxBackoff;
+    private final long initialBackoff;
+    private final double backoffMultiplier;
+
+    public ExponentialBackoffRetryHandler() {
+      this(10000, 500, 1.5);
```

Comment:
retry for 10 seconds, at 500, 750, 1125, 1687, 2531, 3796 ms intervals

Reply:
> retry for 10 seconds, at 500, 750, 1125, 1687, 2531, 3796 ms intervals

would be even better as a code comment!

Reply:
Can we make this configurable via a property?